admin/preprints/serializers.py

```diff
@@ -0,0 +1,29 @@
+from website.project.taxonomies import Subject
+from admin.nodes.serializers import serialize_node
+
+
+def serialize_preprint(preprint):
+
+    return {
+        'id': preprint._id,
+        'date_created': preprint.date_created,
+        'modified': preprint.date_modified,
+        'provider': preprint.provider,
+        'node': serialize_node(preprint.node),
+        'is_published': preprint.is_published,
+        'date_published': preprint.date_published,
+        'subjects': serialize_subjects(preprint.subjects),
+    }
+
+
+def serialize_subjects(subjects):
+    serialized_subjects = []
+    for subject in subjects:
+        if len(subject) == 1:
```

Comment:
They were stored this way for me, not entirely sure if it was an artifact of how my preprints were created or an oddity in general. I can try to make preprints again through the API and see if this persists.